src/firmware/main/io/dr.c

```diff
@@ -4,11 +4,10 @@
 #include <stdint.h>
 #include <stdio.h>
 
+#include "firmware/main/shared/physics.h"
 #include "io/dsp.h"
 #include "io/encoder.h"
 #include "io/sensors.h"
```

Comment:
same as above
```suggestion
#include "firmware/main/io/sensors.h"
```

Reply:
Similar comment to above, out-of-scope for this PR.